src/aihwkit/simulator/configs/utils.py

```diff
@@ -705,16 +705,39 @@ class DriftParameter(SimpleDriftParameter):
 class MappingParameter(_PrintableMixin):
     """Parameter related to the mapping of logical weights to physical tiles.
 
-    Note:
+    Caution:
 
-        These parameters have only an affect for modules that
+        Some of these parameters have only an affect for modules that
         support mappings.
     """
 
+    digital_bias: bool = True
+    """Whether the bias term is handled by the analog tile or kept in
+    digital.
+
+    Note:
+        Default is having a *digital* bias so that bias values are
+        *not* stored onto the analog crossbar. This needs to be
+        supported by the chip design. Set to False if the analog bias
+        is instead situated on the the crossbar itself (as an extra
+        column)
+
+    Caution:
+        ``digital_bias`` is supported by *all* analog modules.
+    """
+
     max_input_size: int = 512
     """Maximal size of the physical in the input dimension.
```

Comment:
Not clear what you mean by physical. Do you mean Maximal size of the physical tile length in the input dimension?